### PR TITLE
olya_task3_calculating_squares_of_numbers

### PR DESCRIPTION
**Pull request by olyapetryshyn**

### CODE REVIEW

File: olya_petryshyn/calculating_squares_of_numbers.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"""
+Calculating squares of numbers in a large list without memory errors.
+"""
+squared_list_generator = (x*x for x in range(100000000000))
```
[the remark this answers is not in the document]
Comment (olyapetryshyn) [reply olya_petryshyn/calculating_squares_of_numbers.py]: Just checked. Faced memory error there.

[the remark this answers is not in the document]
Comment (pivancy) [reply olya_petryshyn/calculating_squares_of_numbers.py]: Please try to fix this issue somehow.
Your solution should work fine under both versions: 2.X, 3.X

[the remark this answers is not in the document]
Comment (olyapetryshyn) [reply olya_petryshyn/calculating_squares_of_numbers.py]: fixed.

**Comment by pivancy** [comment]
Do you really understand why and how it works now?

👍

**Pull request merged**